serverless/src/submitHarmonyOrder/bboxToPolygon.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Not sure we need to do it now, but this seems like it'd be useful as a shared util.